laravel/webpack-config.js

```diff
@@ -9,6 +9,6 @@ module.exports =
 
     output:
     {
-        filename: './bundle.js'
+        filename: './bundle.js' ;
```

Comment:
Why was this modified?

Reply:
I ran it through a linter and the linter said it needed a ;  at least that's what I think happened